nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -135,29 +89,17 @@
             .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_MAXIMUM_MSG_SIZE = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_MSG_SIZE = new PropertyDescriptor.Builder()
```

Comment:
I was just propagating property value to it's corresponding property in SMTPServer, so IMO if server doesn't care why should we? Note that with the refactoring we never write a message into memory so even if you try to send TB it will be successful.
If on the other hand we are to provide some type of enforcement on the NiFi side , then we should completely disassociate this property from SMTPServer or have some other property to eliminate the confusion.